tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/HttpClient.scala

```diff
@@ -36,6 +36,11 @@ class HttpClient private (baseUrl: Uri, httpExt: HttpExt)(implicit as: ActorSyst
   def apply(req: HttpRequest): Task[HttpResponse] =
     Task.deferFuture(httpExt.singleRequest(req))
 
+  def head(url: Uri, identity: Identity)(assertResponse: HttpResponse => Assertion): Task[Assertion] = {
+    val req = HttpRequest(HEAD, s"$baseUrl$url", headers = identityHeader(identity).toList)
+    Task.deferFuture(httpExt.singleRequest(req)).map(assertResponse)
+  }
```

Comment:
I ended up with this new method. I have left in the commented out initial approach I had. Any thoughts welcome